runio.js

```diff
@@ -86,11 +86,19 @@ Our community prepared some valuable recipes for setting up CI systems with Code
     // generate documentation for helpers outside of main repo
     console.log('Building @codecepjs/detox helper docs');
     const helper = 'Detox';
+    replaceInFile(`node_modules/@codeceptjs/detox-helper/${helper}.js`, (cfg) => {
+      cfg.replace(/CodeceptJS.LocatorOrString/g, 'string | object');
```

Comment:
Why is this feint here?

Whe replace on 'string | object', and then back again?

What will be with not `LocatorOrString`, but  `'string | object'` types?